File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Exercise/08_Lunch_Break.js
function lunchBreak(input) {
    let show = input[0];
    let showDuration = Number(input[1]);
    let breakDuration = Number(input[2]);

    let lunchTime = breakDuration / 8;
    let restTime = breakDuration / 4;

    breakDuration -= lunchTime + restTime;

    if (breakDuration >= showDuration) {
        console.log(`You have enough time to watch ${show} and left with ${Math.ceil(breakDuration - showDuration)} minutes free time.`);
    } else {
        console.log(`You don't have enough time to watch ${show}, you need ${Math.ceil(showDuration - breakDuration)} more minutes.`);
    }
}

// lunchBreak(["Teen Wolf", "48", "60"]);

File: Python_Courses/Python_Basics/Python_PB_2024_01/02_Conditional_Statements_Exercise/08_Lunch_Break.py
from math import ceil

series = input()
series_length = int(input())
break_length = int(input())

# Времето за обяд ще бъде 1/8 от времето за почивка,
lunch_duration = break_length * 1 / 8
# времето за отдих ще бъде 1/4 от времето за почивка.
rest_duration = break_length * 1 / 4

time_remaining = break_length - (lunch_duration + rest_duration)

if time_remaining >= series_length:
    time_left = time_remaining - series_length
    print(f'You have enough time to watch {series} and left with {ceil(time_left)} minutes free time.')
else:
    time_needed = series_length - time_remaining
    print(f'You don\'t have enough time to watch {series}, you need {ceil(time_needed)} more minutes.')
